incubator/kafka/values.yaml

```diff
@@ -143,52 +143,115 @@ persistence:
-      #   - pattern : kafka.server<type=ReplicaFetcherManager, name=MaxLag, clientId=(.+)><>Value
-      #   - pattern : kafka.server<type=BrokerTopicMetrics, name=(BytesInPerSec|BytesOutPerSec|MessagesInPerSec), topic=(.+)><>OneMinuteRate
-      #   - pattern : kafka.server<type=KafkaRequestHandlerPool, name=RequestHandlerAvgIdlePercent><>OneMinuteRate
-      #   - pattern : kafka.server<type=Produce><>queue-size
-      #   - pattern : kafka.server<type=ReplicaManager, name=(PartitionCount|UnderReplicatedPartitions)><>(Value|OneMinuteRate)
-      #   - pattern : kafka.server<type=controller-channel-metrics, broker-id=(.+)><>(.*)
-      #   - pattern : kafka.server<type=socket-server-metrics, networkProcessor=(.+)><>(.*)
-      #   - pattern : kafka.server<type=Fetch><>queue-size
-      #   - pattern : kafka.server<type=SessionExpireListener, name=(.+)><>OneMinuteRate
-      #   - pattern : kafka.controller<type=KafkaController, name=(.+)><>Value
-      #   - pattern : java.lang<type=OperatingSystem><>SystemCpuLoad
-      #   - pattern : java.lang<type=Memory><HeapMemoryUsage>used
-      #   - pattern : java.lang<type=OperatingSystem><>FreePhysicalMemorySize
+    ## Port jmx-exporter exposes Prometheus format metrics to scrape
+    port: 5556
 
+    resources: {}
+      # limits:
+      #   cpu: 200m
+      #   memory: 1Gi
+      # requests:
+      #   cpu: 100m
+      #   memory: 100Mi
+
+  ## Prometheus Kafka Exporter: exposes complimentary metrics to JMX Exporter
   kafka:
     enabled: false
 
-    # The image to use for the metrics collector
+    ## The image to use for the metrics collector
     image: danielqsj/kafka-exporter
 
-    # The image tag to use for the metrics collector
+    ## The image tag to use for the metrics collector
     imageTag: v1.0.1
 
+    ## Interval at which Prometheus scrapes metrics, note: only used by Prometheus Operator
+    interval: 10s
+
+    ## Port kafka-exporter exposes for Prometheus to scrape metrics
+    port: 9308
+
+    ## Resource limits
+    resources: {}
+#      limits:
+#        cpu: 200m
+#        memory: 1Gi
+#      requests:
+#        cpu: 100m
+#        memory: 100Mi
+
+  operator:
+    ## Are you using Prometheus Operator?
+    enabled: false
+
+    serviceMonitor:
+      # Namespace Prometheus is installed in
+      namespace: monitoring
+
+      ## Defaults to whats used if you follow CoreOS current instructions:
+      ## https://github.com/coreos/prometheus-operator/tree/master/helm#tldr
+      ## helm repo add coreos https://s3-eu-west-1.amazonaws.com/coreos-charts/stable/
+      ## helm install coreos/prometheus-operator --name prometheus-operator --namespace monitoring
+      ## helm install coreos/kube-prometheus --name kube-prometheus --namespace monitoring
+      ## https://github.com/coreos/prometheus-operator/blob/master/helm/prometheus/templates/prometheus.yaml#L65
+      ## https://github.com/coreos/prometheus-operator/blob/master/helm/kube-prometheus/values.yaml#L298
```

Comment:
Small nit: Do these instructions exist in a separate place already? If so, would linking to them be better? I would worry that duplicating instructions for installing an operator here could easily get out of date